src/playground/services/session/login/endpoint.clj

```diff
@@ -24,7 +24,7 @@
         session (:session request)]
     (if (logic/check-password password (:password (password-by-username request username)))
       (let [next-url (get-in request [:query-params :next] "/")
-            updated-session (assoc session :identity {:username username :password password})]
+            updated-session (assoc session :identity {:username username :password password :role (logic/role username)})]
```

Comment:
I believe that `:password` in the `:identity` map must be the hashed password `(buddy.hashers/derive password) is that correct?